### ISSUE TITLE
[make install error] command_mode.h:27:10: fatal error: curses.h: No such file or directory

### ISSUE THREAD

**Issue by FanMclaine** [cmus/cmus#1203]
---

1) `uname -a`:
```
Linux lenovo-ideapad 4.19.0-13-686-pae #1 SMP Debian 4.19.160-2 (2020-11-28) i686 GNU/Linux
```

2) `cmus --version`
```
new version? ( i forgot what it is but I just git cloned like 5 minutes ago)
```
```
pi@lenovo-ideapad:~ $ cmus --version
bash: cmus: command not found
```

3) `cmus --plugins`:
```
nothing
```
![2022-06-17-201305_1024x576_scrot](https://user-images.githubusercontent.com/66262586/174297434-6335bef8-c329-46e0-8afc-7459a8d24fb0.png)

**Comment by gavtroy** [comment]
Re-run ./configure. If it can't find ncurses you might need to reinstall it.

**Comment by pgaskin** [comment]
Probably fixed by #1349 (9d87ac8fae14ca0b5ea1574bd757a3ef0c3e4a9f).
